klab/settings.py.dev

```diff
@@ -1,12 +1,12 @@
 from settings_common import *
 
-INSTALLED_APPS = INSTALLED_APPS + ('django_nose', 'debug_toolbar')
+INSTALLED_APPS = INSTALLED_APPS + ('django_nose',) #, 'debug_toolbar')
```

Comment:
these are you own local settings, it is better you move them in a file and not add it to the repo like settings.py.local